docs/SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If there are any vulnerabilities in **PHPDoc Parser**, don't hesitate to _report them_.
+
+1. Use any of the [private contact addresses](https://github.com/RightCapitalHQ/phpdoc-parser#support).
```

Comment:
We may add a security policy(e.g. [SECURITY.md](https://github.com/pnpm/pnpm/blob/main/SECURITY.md)) that enables users to submit security advisories via GitHub?

Reply:
I just added the email address to the doc. (I actually updated the doc in the local and forgot to commit it.)